wp-cli/vip-migrations.php

```diff
@@ -147,6 +147,54 @@ public function validate_attachments( $args, $assoc_args ) {
 		WP_CLI\Utils\write_csv( $file_descriptor, $output );
 		fclose( $file_descriptor );
 	}
+
+	/**
+	 * Import user meta attributes from a CSV file.
+	 *
+	 * @subcommand import-user-meta
+	 * @synopsis <file> [--user_key=<userlogin>]
+	 */
+	function import_user_meta( $args, $assoc_args ) {
+		$filename = $args[0];
+		$user_key = $assoc_args['user_key'] ?? 'user_login';
+
+		if ( ! file_exists( $filename ) ) {
+			WP_CLI::error( sprintf( 'Missing file: %s', $filename ) );
+		}
+
+		foreach ( new \WP_CLI\Iterators\CSV( $filename ) as $user_data ) {
+			$user_data = array_values( $user_data ); // Strip useless array keys.
```

Comment:
Makes sense. Should we update the docs to mention that this is meant to be used with a CSV generated from the WP.com CLI command?